WikiTemplate/Desktop/wikitemplate.md

```diff
@@ -193,7 +193,7 @@
   - [ ] `CertificateRevocation`
   - [ ] `cffkpbalmllkdoenhmdmpbkajipdjfam`: `rs-ABPFilterParserData.dat` & `regional_catalog.json` (AdBlock)
   - [ ] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
-  - [ ] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
+  - [ ] `jicbkmdloagakknpihibphagfckhjdih`: `speedreader-updater.dat`
```

Comment:
I have no idea where this change snuck in from, but are they ok to sneak in, so I don't have to rebase/fix merge conflicts yet again, @LaurenWags ?

Reply:
Is it accurate? Not sure where this one came from either. FWIW I'm not seeing this one on a fresh install of 1.68.x - so we should be careful about adding something that is wrong.